### PR TITLE
Update http header `User-Agent` in k8s and swarm  discovery.

### PR DESCRIPTION
**Pull request by johncming**
It's common that the http header `User-Agent` has the format below:

```
fmt.Sprintf("Prometheus/%s", version.Version)
```

For example:

```
notifier/notifier.go
59:var userAgent = fmt.Sprintf("Prometheus/%s", version.Version)
580:	req.Header.Set("User-Agent", userAgent)

scrape/scrape.go
532:var userAgentHeader = fmt.Sprintf("Prometheus/%s", version.Version)
542:		req.Header.Set("User-Agent", userAgentHeader)

discovery/digitalocean/digitalocean.go
100:		godo.SetUserAgent(fmt.Sprintf("Prometheus/%s", version.Version)),

storage/remote/client.go
43:var userAgent = fmt.Sprintf("Prometheus/%s", version.Version)
```

This pr is for using the format.

### CODE REVIEW

**Comment by roidelapluie** [comment]
Let's wait for #7604 to be merged back to master

**Comment by johncming** [comment]
> Let's wait for #7604 to be merged back to master

ok.

**Comment by roidelapluie** [comment]
As that #7604  is for the release branch, let's make the merge back to master easier, that is why.

**Comment by johncming** [comment]
> As that #7604 is for the release branch, let's make the merge back to master easier, that is why.

Ok, I see. This PR is not for a bug, so take your time.

File: discovery/dockerswarm/dockerswarm.go [hunk 1]
```diff
@@ -121,6 +124,9 @@ func NewDiscovery(conf *SDConfig, logger log.Logger) (*Discovery, error) {
 				Timeout:   time.Duration(conf.RefreshInterval),
 			}),
 			client.WithScheme(conf.url.Scheme),
+      client.WithHTTPHeaders(map[string]string{
```
Comment (roidelapluie) [inline discovery/dockerswarm/dockerswarm.go:127]: identation is off

**Comment by roidelapluie** [comment]
You removed swarm?

**Comment by johncming** [comment]
> You removed swarm?

Yes, I see that your comment `identation is off`.  

Need I back swarm?

**Comment by roidelapluie** [comment]
My comment was that you needed to run gofmt :))

**Comment by johncming** [comment]
> My comment was that you needed to run gofmt :))

Sorry, I misunderstood.  :))

File: discovery/kubernetes/kubernetes.go [hunk 2]
```diff
@@ -48,6 +50,8 @@ const (
 	presentValue     = model.LabelValue("true")
 )
 
+var userAgent = fmt.Sprintf("Prometheus/%s", version.Version)
```
Comment (roidelapluie) [inline discovery/kubernetes/kubernetes.go:53]: Can you add the var in the block below?

Comment (roidelapluie) [reply discovery/kubernetes/kubernetes.go:53]: Did you see this?

Comment (johncming) [reply discovery/kubernetes/kubernetes.go:53]: OK.

**Comment by roidelapluie** [comment]
Thanks!

**Pull request merged**